src/tests/ftest/util/environment_utils.py

```diff
@@ -109,6 +109,9 @@ class TestEnvironment():
         'daos_prefix': 'DAOS_TEST_PREFIX',
         'agent_user': 'DAOS_TEST_AGENT_USER',
         'systemd_library_path': 'DAOS_TEST_SYSTEMD_LIBRARY_PATH',
+        'control_config': 'DAOS_TEST_CONTROL_CONFIG',
+        'client_config': 'DAOS_TEST_CLIENT_CONFIG',
```

Comment:
client_config? Maybe directly call it agent_config?

Reply:
I debated that one, but I think I agree with you.